File: tools/conf/output/go/StockConfig.go
package conf

import (
	//"bufio"
	"encoding/json"
	"fmt"
	//"io"
	"io/ioutil"
	"os"
	//"strings"
)

type StockConfigElem struct {
	Id int
	Ukey int
	Market_id int
	Major_type int
	Minor_type int
	Market_code int
	Market_abbr string
	Chinese_name string
	English_name string
	List_date string
	Delist_date string
	Currency_id string
	Jy_code string
	Wind_code string
	Input_code string
	Last_update string
	Show int
	
}

type StockConfig0 struct {
	Dic map[int]*StockConfigElem
	Arr []*StockConfigElem
}

var (
	StockConfig StockConfig0
)

func loadStockConfig(path string) {
	file, err := os.Open(path)
	defer func() {
		if file != nil {
			file.Close()
		}
	}()
	if err != nil {
		panic(fmt.Sprint("load config error, path=", path))
	}

	StockConfig.Dic = make(map[int]*StockConfigElem)
	StockConfig.Arr = make([]*StockConfigElem, 0, 32)

	bt,err  :=  ioutil.ReadAll(file)
	json.Unmarshal(bt, &StockConfig.Arr)
	for _,conf := range StockConfig.Arr{
		StockConfig.Dic[conf.Id] = conf
	}
	
}

File: tools/conf/output/ts/StockConfig.ts
class StockConfig {
    private static _dic:Dictionary<StockConfig> = new Dictionary<StockConfig>();
    private static _arr:Array<StockConfig> = new Array<StockConfig>();

	 public Id :number ;
     public Ukey :number ;
     public Market_id :number ;
     public Major_type :number ;
     public Minor_type :number ;
     public Market_code :number ;
     public Market_abbr :string ;
     public Chinese_name :string ;
     public English_name :string ;
     public List_date :string ;
     public Delist_date :string ;
     public Currency_id :string ;
     public Jy_code :string ;
     public Wind_code :string ;
     public Input_code :string ;
     public Last_update :string ;
     public Show :number ;
    

    
    public constructor(){

    }


    public static get Dic():Dictionary<StockConfig>{
        return  StockConfig._dic;
    }
    public static get Arr():Array<StockConfig>{
        return StockConfig._arr;
    }
    public static parse(json:any[]){

        for(var i = 0; i < json.length; i++){
            let config:StockConfig = <StockConfig>json[i];
            StockConfig._arr.push(config);
            StockConfig._dic.add(config.Id,config);
        }
    }
}